src/utils/device.jl

```diff
@@ -22,6 +23,8 @@ send_to_device(
             <:Any,
             <:Base.ReshapedArray{<:Any,<:Any,<:SubArray{<:Any,<:Any,<:CircularArrayBuffer}},
         },
+        SubArray{<:Any,<:Any,<:ElasticArray,<:Any,<:Any},
+        ElasticArray,
```

Comment:
Is it necessary here? I remember that ElasticArray supports CUDA.jl but I may be wrong. Away from computer so I can't verify it…

Reply:
I'm not sure.
I got an error, and filed an issue https://github.com/JuliaArrays/ElasticArrays.jl/issues/30 .

Reply:
For `ElasticArray` with `data` of `CuArray`, there's no need to create a new copy with `CuArray(x)`. But I think that's a rare case here.